Intel_SighPP_CMake/Intel_SighPP_CMake/src/priority_lib/smart_priority.cpp

```diff
@@ -15,7 +19,18 @@ void smart_priority::determine_location_markers() {
 	in_right = mid_w + fifth_w;
 	out_left = 0 + fifth_w;
 	out_right = size_w - fifth_w;
-	above = 0 + eighth_h;
+	above = size_h - eighth_h;
+	SPDLOG_INFO("mid_w height = {}", mid_w);
+
+	SPDLOG_INFO("in_left height = {}", in_left);
+
+	SPDLOG_INFO("in_right height = {}", in_right);
+
+	SPDLOG_INFO("out_left height = {}", out_left);
+
+	SPDLOG_INFO("out_right height = {}", out_right);
+
+	SPDLOG_INFO("above height = {}", above);
```

Comment:
Maybe these could be put into one statement, but it probably alright cause this only gets printed once right?